code/game/objects/buckling.dm

```diff
@@ -11,7 +11,7 @@
 /atom/movable/attack_hand(mob/living/user)
 	. = ..()
 	if(can_buckle && has_buckled_mobs())
-		if(buckled_mobs.len > 1)
+		if(LAZYLEN(buckled_mobs) > 1)
```

Comment:
`buckled_mobs` isn't a lazy list, so don't do this.

Reply:
Done